docs/contributing.md

```diff
@@ -14,7 +16,64 @@ Nano has a relatively small community of developers and only a handful of develo
+
+Being that Nano is a protocol, to help increase use of the network, it is important to attract entrepeneurs, developers and enthusiasts to help build and improve it. Nano, much like the internet at first, is still hard to use for the vast majority of consumers and businesses. Thus we need builders to make it more accessible so that everyone can use it.
+
+#### Target Audience
+
+##### Speculators / Holders
+
+Why: For Nano to sustain high real world payment volume it needs to be worth something and have enough liquidity. Thus before there is utility, we need speculators and holders. Due to the positive feedback loop of its network effects, the more who are invested, the more attention the project gets, the more businesses are interested in integrating Nano payments, which then attracts even more investors, companies, developers and exchanges continuing the feedback loop.
+
+Pitch: Nano is a strongly undervalued cryptocurrency. Both as a medium of exchange and store of value, it is fundamentally the best there is in the space. As a medium of exchange, Nano being instant, feeless and scalable means that it is hard to imagine something better. As a store of value, Nano's zero inflation, eco-friendliness and game theory leading to ever more decentralization over time make it potentially the best store of value. However, many cryptocurrencies have higher market caps than Nano. If one analyses any other cryptocurrency that claims to be a store of value or medium of exchange, they'll find that Nano is fundamentally better. Because of this, it stands to reason that as the market gets more mature and rational, Nano's recognition and price will increase.
+
+##### Open source developers
+
+##### Exchanges
+
+Why: A currency becomes more useful and valuable when more people own it. The easier it is for people to exchange into Nano, the better. At the same time, the easier it is to sell Nano, the more likely it is that people will use accept Nano payments, knowing they can exchange into fiat easily if they so desire. More exchange listings also leads to a more decentralized network, as the Nano supply becomes more spread out.
+
+Pitch: Nano is interesting both from a direct business and a marketing perspective. In terms of direct business, Nano has a sizeable and growing trading volume. It has clear usecases in being used for payments and remittances. Many of those receiving Nano want to exchange into fiat or other cryptocurrencies. Additionally, many new Nano holders are new to crypto as a whole, and are looking for an exchange to buy Nano on. Nano also has a large and vocal community. Any exchange listing is broadly shared on all sorts of social mediums (Twitter, Reddit) and is one of the best forms of free marketing available.
```

Comment:
Similar to the pitch/benefits for payment processors, it's worth pointing out that Nano's potential to be utilized as a frictionless medium of exchange will lead to more trading volume, which means more revenue for exchanges.